ccmlib/scylla_node.py

```diff
@@ -374,7 +374,6 @@ def start(self, join_ring=True, no_wait=False, verbose=False,
                 except Exception as msg:
                     print("{}. Looking for offending processes...".format(msg))
                     for proc in psutil.process_iter():
-                        print  proc.cmdline()
```

Comment:
python3 related?

Reply:
yes, it's breaks python, and it's not actully need (I think it's my mistake in older PR)